packages/realm-server/migrations/1726147649121_seed-permissions.js

```diff
@@ -0,0 +1,42 @@
+exports.up = (pgm) => {
+  // Add any specific users. If there are any new users to add after this
+  // migration has run then please open a new PR with the user(s) to add for the
+  // specific environment
+  switch (process.env.REALM_SENTRY_ENVIRONMENT) {
+    case 'staging':
+      pgm.sql(
+        `INSERT INTO realm_user_permissions (realm_url, username, read, write)
+         VALUES
+           ('https://realms-staging.stack.cards/seed/', 'habdelra1', true, true)`,
+      );
+      break;
+    case 'production':
+      pgm.sql(
+        `INSERT INTO realm_user_permissions (realm_url, username, read, write)
+         VALUES
+           ('https://app.boxel.ai/seed/', 'habdelra1', true, true)`,
+      );
+      break;
```

Comment:
let me know if you want any other users to be added to this list

Reply:
`lukemelia` please

Reply:
`tintinthong` pls

Reply:
@tintinthong I don't see a user in production with the username `tintinthong`. can you please register one?